app/profile.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+use Carbon\Carbon;
+class profile extends Model
+{
+    protected $fillable = array('DOB');
```

Comment:
Maintain consistency in naming, should be lowercase dob or use the full meaning date_of_birth